### PR TITLE
feat: change merkl typings

### PR DESCRIPTION
**Pull request by greedythib**

### CODE REVIEW

File: src/types/merkl.ts [hunk 1]
```diff
@@ -81,6 +81,8 @@ export type DistributionDataType<T extends AMMType> = {
   amm: AMMType;
   amount: number; // Amount distributed
   breakdown?: { [origin in RewardOrigin<T>]?: number }; // rewards earned breakdown
+  // distributionAPRs: { [`APR (rewards/pool TVL)`]: number; [description: string]: number };
```
Comment (sogipec) [inline src/types/merkl.ts:84]: can be optional no? For some pools we don't have the price of the reward token

Comment (greedythib) [reply src/types/merkl.ts:84]: in this case {} will be enough

Comment (greedythib) [reply src/types/merkl.ts:84]: exactly

Comment (sogipec) [reply src/types/merkl.ts:84]: And it won't do empty tooltips or so in the app?

Comment (sogipec) [reply src/types/merkl.ts:84]: Where you believe there is a breakdown but there is none?

Comment (greedythib) [reply src/types/merkl.ts:84]: this is only used for our own merkl distributions in the `earn` page

Comment (sogipec) [reply src/types/merkl.ts:84]: But didn't we say that what the Earn page shouldn't have any specific logic?

Comment (greedythib) [reply src/types/merkl.ts:84]: yes but I have to add aprs per distributions anyway (`earn` page -> we don't have the choice, `merkl` -> will enable more granularity in the future)

**Review by sogipec — APPROVED** [review]

**Pull request closed without merging**